js/app.js

```diff
@@ -169,6 +169,9 @@ var myApp = (function (_config) {
                 case 'saveAnnotationRecord':
                     viewer.saveAnatomySVG(data);
                     break;
+                // case 'discardAnnotationChanges':
```

Comment:
remove this

Reply:
So this should be added when I do the commit for the Undo issue?

Reply:
Yup.

Reply:
Removed